### PR TITLE
Pr/address field

### PR DESCRIPTION
**Pull request by khanlifts**

### CODE REVIEW

File: packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+    return (
+      <div {... containerProps}>
+        {children}
+      </div>
+    )
+  }
+
+  renderSuggestionsCityContainer = ({containerProps, children}) => {
+    if (this.props.options.isLoading && this.state.cityFieldFocused) {
```
[the remark this answers is not in the document]
Comment (khanlifts) [reply packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js]: containerProps by react-autosuggest need one method per Autosuggest component prop.

File: packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+import ButtonLink from '../../ButtonLink'
+import {
+  StyledLocationEdit,
+  StyledZipInput
+} from './StyledLocationEdit'
+import IconTocco from '../../IconTocco'
+
+export const getMapsAddress = locationInput => {
+  const mapsBaseAddress = `https://www.google.com/maps/search/?api=1&query=`
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js]: hide maps button if the generated link does not make sense. for example if address is empty...

File: packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        locationAcc.push(locationInput[key].display)
+      }
+    }
+    return `${mapsBaseAddress}${locationAcc.join('+')}`
+  }
+  
+  return mapsBaseAddress
+}
+
+export const getSuggestions = (value, suggestions) => {
```
[the remark this answers is not in the document]
Comment (khanlifts) [reply packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js]: Those methods don't need props.

File: packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    const inputLength = inputValue.length
+
+    return inputLength === 0 ? [] : suggestions
+  }
+  return []
+}
+
+export const returnGetSuggestion = attr => suggestion => suggestion[attr]
+
+const renderSuggestion = suggestion => {
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js]: what is the purpose of the `cantonString` variable? You rather just insert the hyphen in the template 

```
  return (
    <div>
      { `${suggestion.zip} .... - ${suggestion.canton}`}
    </div>
  )
```

you additional variable just makes things harder to read.
Any as said in a comment before, you still have to problem that if canton is not given you generate a string like:

> 8006 Zürich - / Switzerland


same for country / countryString

**Pull request closed without merging**